src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -105,69 +112,80 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     utils.validate_location(cmd, location)
     resourceClient = _resource_client_factory(cmd.cli_ctx, subscription_id=subscription_id)
 
-    # Check Release Existance
-    release_namespace = get_release_namespace(kube_config, kube_context)
-    if release_namespace is not None:
-        # Loading config map
-        api_instance = kube_client.CoreV1Api(kube_client.ApiClient(configuration))
-        try:
-            configmap = api_instance.read_namespaced_config_map('azure-clusterconfig', 'azure-arc')
-        except Exception as e:  # pylint: disable=broad-except
-            telemetry.set_exception(exception=e, fault_type=consts.Read_ConfigMap_Fault_Type,
-                                    summary='Unable to read ConfigMap')
-            raise CLIError("Unable to read ConfigMap 'azure-clusterconfig' in 'azure-arc' namespace: %s\n" % e)
-        configmap_rg_name = configmap.data["AZURE_RESOURCE_GROUP"]
-        configmap_cluster_name = configmap.data["AZURE_RESOURCE_NAME"]
-        if connected_cluster_exists(client, configmap_rg_name, configmap_cluster_name):
-            if (configmap_rg_name.lower() == resource_group_name.lower() and
-                    configmap_cluster_name.lower() == cluster_name.lower()):
-                # Re-put connected cluster
-                public_key = client.get(configmap_rg_name,
-                                        configmap_cluster_name).agent_public_key_certificate
-                cc = generate_request_payload(configuration, location, public_key, tags)
-                try:
-                    return sdk_no_wait(no_wait, client.create, resource_group_name=resource_group_name,
-                                       cluster_name=cluster_name, connected_cluster=cc)
-                except CloudError as ex:
-                    telemetry.set_exception(exception=ex, fault_type=consts.Create_ConnectedCluster_Fault_Type,
-                                            summary='Unable to create connected cluster resource')
-                    raise CLIError(ex)
+    # If this is direct access token based onboarding, this will be done by connect-agent itself.
+    # Therefore, this is done only if it is not token_based_onboarding.
+    if (token_based_onboarding is None) or (token_based_onboarding == "false") or (not token_based_onboarding):
```

Comment:
Why do you have three checks here? The not operator works for bool false as well as None values of the variables

Reply:
Okay, will change that.